src/test/scala/org/scalabridge/sitegen/StaticSiteGeneratorSuite.scala

```diff
@@ -24,4 +26,11 @@ class StaticSiteGeneratorSuite extends ScalaCheckSuite {
       )
     }
   }
+
+  test("Bold test") {
+    assertEquals(
+      parse("**bold text**", strongParser).map(generateHtml).map(_.render),
+      Right("<strong>bold text</strong>")
+    )
+  }
```

Comment:
If you look at the tests above, you'll see that we can use generators to generate random values, without relying on static examples. Can you implement this test (and perhaps a failure test too, in terms of those?